.github/workflows/criterion_benchs.yml

```diff
@@ -23,7 +22,11 @@ jobs:
           override: true
     - uses: actions/checkout@v3
     - name: Run benchmark
-      run: cargo bench --bench all_benchmarks -- --output-format bencher | tee output.txt
+      run: |
```

Comment:
this means that if a new bench is added, it has to be added here?

Reply:
Yes, if it makes sense to run them on here. For example this job is missing the `criterion_metal` benchs that should execute on a different job running on our M2s, and `criterion_field` which benchs nanosecond ops between field elements, whose results get totally drowned with the CI's generated noise (check out [here](https://lambdaclass.github.io/lambdaworks/bench/)).

Reply:
makes sense, thx!